src/components/Table.tsx

```diff
@@ -42,7 +43,6 @@ function TableCell(props: TableCellProps) {
 				</a>
 				<Tooltip className="soc-tooltip" id={`${props.index.toString()}-${props.society.symbol}`} delayHide={0} delayShow={0}>
 					<span className="soc-tooltip-name">{props.society.name}</span>
-					<p className="soc-toolip-desc">{props.society.description}</p>
```

Comment:
?

Reply:
in the thread we discussed that tooltip will only show the name and there will be a modal for the desc?

Reply:
Is a tooltip useful if we have a modal?